redbpf/src/lib.rs

```diff
@@ -502,7 +502,7 @@ impl Program {
     }
 
     fn with_btf(kind: &str, name: &str, code: &[u8], btf: &BTF) -> Result<Program> {
-        let code = zero::read_array(code).to_vec();
+        let code = unsafe { zero::read_array_unsafe(code) }.to_vec();
```

Comment:
What's the motivation behind dropping checks during this step?

Reply:
`zero` crate allows to use safe methods only for structs which implements (unsafe) trait `Pod`. `bpf-sys` crate implemented this trait for some bindings structs. But now it can't, because both struct and trait are from foreign creates. So `unsafe` moved from `impl` to call site.

Reply:
Ah, I forgot about this! Makes sense.